velox/common/memory/SharedArbitrator.cpp

```diff
@@ -396,9 +396,11 @@ uint64_t SharedArbitrator::reclaim(
     uint64_t targetBytes) noexcept {
   const uint64_t oldCapacity = pool->capacity();
   uint64_t freedBytes{0};
+  uint64_t reclaimDurationUs{0};
   try {
     freedBytes = pool->shrink(targetBytes);
     if (freedBytes < targetBytes) {
+      MicrosecondTimer reclaimTimer(&reclaimDurationUs);
```

Comment:
Shall we cover entire reclaim block including the failure path that do abort? Thanks!